$NetBSD: patch-aa,v 1.9 2013/11/24 04:01:07 dholland Exp $

- honor PREFIX, PKG_SYSCONFDIR, PKGINFODIR
- XXX: should but does not honor VARBASE
- use /var/spool/fido rather than /var/spool/fidogate (why?)
- use /var/news rather than /var/lib/news and/or /var/spool/news
- use pkgsrc perl
- set SENDMAIL_CF_DIR (XXX: it is no longer correct)
- use pkgsrc compiler and yacc
- don't build with -g by default
- use pkgsrc install tools
- handle setuid perms with SPECIAL_PERMS rather than requiring root to build

--- config.make.orig	2003-04-15 18:45:20.000000000 +0000
+++ config.make
@@ -56,13 +56,13 @@
 #
 
 # variable parameters, can be changed at run-time, DO NOT DELETE ANYTHING!!!
-DEFAULT_V_CONFIGDIR	= /etc/fidogate
-DEFAULT_V_LIBDIR	= /usr/lib/fidogate
-DEFAULT_V_BINDIR	= $(DEFAULT_V_LIBDIR)/bin
-DEFAULT_V_LOGDIR	= /var/log/fidogate
-DEFAULT_V_VARDIR	= /var/lib/fidogate
-DEFAULT_V_LOCKDIR	= /var/lock/fidogate
-DEFAULT_V_SPOOLDIR	= /var/spool/fidogate
+DEFAULT_V_CONFIGDIR	= @PKG_SYSCONFDIR@/fidogate
+DEFAULT_V_LIBDIR	= @PREFIX@/lib/fidogate
+DEFAULT_V_BINDIR	= @PREFIX@/lib/fidogate/bin
+DEFAULT_V_LOGDIR	= /var/log/fido
+DEFAULT_V_VARDIR	= /var/spool/fido
+DEFAULT_V_LOCKDIR	= /var/spool/fido/lock
+DEFAULT_V_SPOOLDIR	= /var/spool/fido
 DEFAULT_V_BTBASEDIR	= /var/spool/bt
 DEFAULT_V_INBOUND	= $(DEFAULT_V_BTBASEDIR)/in
 DEFAULT_V_PINBOUND	= $(DEFAULT_V_BTBASEDIR)/pin
@@ -83,8 +83,8 @@ DEFAULT_V_CHARSETMAP	= %L/charset.bin
 
 # fixed parameters, DO NOT DELETE ANYTHING!!!
 # INN (Redhat 5.2)
-DEFAULT_F_NEWSETCDIR	= /etc/news
-DEFAULT_F_NEWSVARDIR	= /var/lib/news
+DEFAULT_F_NEWSETCDIR	= /var/news/etc
+DEFAULT_F_NEWSVARDIR	= /var/news
 # INN 1.7
 #DEFAULT_F_NEWSLIBDIR	= /usr/lib/news
 #DEFAULT_F_NEWSSPOOLDIR	= /var/spool/news
@@ -92,11 +92,11 @@ DEFAULT_F_NEWSVARDIR	= /var/lib/news
 #DEFAULT_F_NEWSLIBDIR	= /usr/lib
 #DEFAULT_F_NEWSSPOOLDIR	= /var/spool/news/articles
 # INN 2.3.4 on RedHat 9
-DEFAULT_F_NEWSLIBDIR	= /usr/lib/news/lib
-DEFAULT_F_NEWSSPOOLDIR	= /var/spool/news/articles
+DEFAULT_F_NEWSLIBDIR	= @PREFIX@/inn/lib
+DEFAULT_F_NEWSSPOOLDIR	= /var/news/spool/articles
 # ifcico-3.0.cm.alpha-4.1
 #DEFAULT_F_IFMAILDIR     = /usr/lib/ifmail
-DEFAULT_F_IFMAILDIR     = /usr/lib/ifcico
+DEFAULT_F_IFMAILDIR     = @PREFIX@/sbin
 
 # old-style config
 #DEFAULT_F_CONFIG_GATE	= %C/gate.conf
@@ -149,18 +149,15 @@ DEFAULT_A_UUINBOUND	= %U
 
 
 # The perl interpreter used by subst.pl
-PERL			= /usr/bin/perl
+PERL			= @PERL@
 
 # Directory with sendmail m4 configuration files
-# RedHat 6.x
-#SENDMAIL_CF_DIR		= /usr/lib/sendmail-cf
-# RedHat 7.x
-SENDMAIL_CF_DIR		= /usr/share/sendmail-cf
+SENDMAIL_CF_DIR		= /usr/share/sendmail
 
 # Directories for installing documentation, not used by subst.pl
-INFODIR			= /usr/info
-HTMLDIR			= /html/fidogate
-HTMLLOGDIR		= /html/log
+INFODIR			= @PREFIX@/@PKGINFODIR@
+HTMLDIR			= @PREFIX@/share/doc/fidogate
+HTMLLOGDIR		= @PREFIX@/share/fidogate
 
 # RedHat RPM related dirs
 RPMBASEDIR	= /usr/src/redhat
@@ -200,20 +197,17 @@ GROUP		= news
 # install permissions
 PERM_PROG	= 755
 PERM_DATA	= 644
-PERM_SETUID	= 4755
+PERM_SETUID	= 4111
 PERM_DIR	= 755
 
 # C compiler / flags
-CC		= gcc
-# YACC		= yacc					# Use yacc, not bison
-YACC		= bison -y
+YACC		= yacc					# Use yacc, not bison
 AR		= ar
 # RANLIB	= @echo >/dev/null			# No ranlib
 RANLIB		= ranlib
 # RANLIB	= ar s					# OS2
 
 # DEBUG		= -O2
-DEBUG		= -g
 
 INCLUDE		= -I$(TOPDIR) -I$(TOPDIR)/src/include
 
@@ -242,10 +236,10 @@ INSTALL		= install
 # MSDOS, OS2, WIN32
 # EXE		= .exe
 EXE		=
-INSTALL_PROG	= $(INSTALL) -c -g $(GROUP) -o $(OWNER) -m $(PERM_PROG)
-INSTALL_DATA	= $(INSTALL) -c -g $(GROUP) -o $(OWNER) -m $(PERM_DATA)
-INSTALL_SETUID	= $(INSTALL) -c -g $(GROUP) -o $(OWNER) -m $(PERM_SETUID)
-INSTALL_DIR	= $(INSTALL) -g $(GROUP) -o $(OWNER) -m $(PERM_DIR) -d
+INSTALL_PROG	= $(BSD_INSTALL_SCRIPT)
+INSTALL_DATA	= $(BSD_INSTALL_DATA)
+INSTALL_SETUID	= $(BSD_INSTALL_PROGRAM)
+INSTALL_DIR	= $(BSD_INSTALL_DATA_DIR) -g $(GROUP) -o $(OWNER) -m $(PERM_DIR) -d
 # extra prefix for installation
 PREFIX		=
 
